db/migrate/20210709205410_change_qb_object_subtype_to_disbursement_type_on_transaction.rb

```diff
@@ -0,0 +1,15 @@
+class ChangeQbObjectSubtypeToDisbursementTypeOnTransaction < ActiveRecord::Migration[5.2]
+  def up
+    Accounting::Transaction.where(qb_object_subtype: "Cash").update_all(qb_object_subtype: :other)
+    Accounting::Transaction.where(qb_object_subtype: "Check").update_all(qb_object_subtype: :check)
+    rename_column :accounting_transactions, :qb_object_subtype, :disbursement_type
+    change_column_default :accounting_transactions, :disbursement_type, :other
+  end
+
+  def down
+    Accounting::Transaction.where(disbursement_type: :other).update_all(disbursement_type: "Cash")
+    Accounting::Transaction.where(disbursement_type: :check).update_all(disbursement_type: "Check")
+    rename_column :accounting_transactions, :disbursement_type, :qb_object_subtype
+    change_column_default :accounting_transactions, :qb_object_subtype, :nil
+  end
+end
```

Comment:
Nicely done on the migration!

Reply:
One small thing: the default value should be a string rather than a symbol. And I think on the `down` migration you want `nil` rather than `:nil` or `"nil"`...

Reply:
I am curious why we're adding a default here without adding a null constraint. Does this value apply to all transactions or only those of a certain type? If the latter, I think it's more appropriate to keep allowing nulls and remove the default...